src/api/tmdb-api.js

```diff
@@ -1 +1,2 @@
-export const path = 'https://api.themoviedb.org/3'
\ No newline at end of file
+export const path = 'https://api.themoviedb.org/3'
+export const pathFooPoster = 'https://image.tmdb.org/t/p/original'
```

Comment:
What does it mean `Foo`? Probably it's typo and should be `For`?
Anyway better naming is `posterPath`

Reply:
I changed this in Movie Page branch